### ISSUE TITLE
Unable to Connect to Server after 1.0.2 update and server restart

### ISSUE THREAD

**Issue by jehoshua02** [Uuvana-Studios/longvinter-windows-client#587]
**Describe the bug**
Unable to connect to server after 1.0.2 update and server restart.

**To Reproduce**
1. In game, search for "Grokenvinter"
2. Click
3. Seems like it's going to connect.
4. Land back on main menu.

**Expected behavior**
Would expect the client to connect to server and see my character in game.

**Screenshots**
NA

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Game Version 1.0.2
 - Steam Version Updated

**Additional context**

I'm running the server via https://github.com/tvandoorn/longvinter-docker-server and have recreated the container.

Output from `docker compose logs -f`

```
gv1  | [2022.03.25-18.42.45:986][773]LogOnlineSession: Warning: STEAM: Server setting ,SNAME_s:Grokenvinter discord.gg/tCMNAhQzrb overflows Steam SetGameTags call
gv1  | [2022.03.25-18.42.45:987][773]LogOnlineSession: Warning: STEAM: Server setting ,ServerName_s:Grokenvinter discord.gg/tCMNAhQzrb overflows Steam SetGameTags call
gv1  | [2022.03.25-18.42.45:988][773]LogOnlineSession: Warning: STEAM: Server setting ,HasPassword_b:false overflows Steam SetGameTags call
gv1  | [2022.03.25-18.42.45:988][773]LogOnlineSession: Warning: STEAM: Server setting ,IsExperimental_b:false overflows Steam SetGameTags call
gv1  | [2022.03.25-18.42.45:988][773]LogOnline: Verbose: OSS: Async task 'FOnlineAsyncTaskSteamUpdateServer bWasSuccessful: 1 Session: GameSession' succeeded in 0.000855 seconds
gv1  | [2022.03.25-18.43.01:035][223]LogOnlineSession: Warning: STEAM: Server setting ,SNAME_s:Grokenvinter discord.gg/tCMNAhQzrb overflows Steam SetGameTags call
gv1  | [2022.03.25-18.43.01:035][223]LogOnlineSession: Warning: STEAM: Server setting ,ServerName_s:Grokenvinter discord.gg/tCMNAhQzrb overflows Steam SetGameTags call
gv1  | [2022.03.25-18.43.01:035][223]LogOnlineSession: Warning: STEAM: Server setting ,HasPassword_b:false overflows Steam SetGameTags call
gv1  | [2022.03.25-18.43.01:035][223]LogOnlineSession: Warning: STEAM: Server setting ,IsExperimental_b:false overflows Steam SetGameTags call
gv1  | [2022.03.25-18.43.01:035][223]LogOnline: Verbose: OSS: Async task 'FOnlineAsyncTaskSteamUpdateServer bWasSuccessful: 1 Session: GameSession' succeeded in 0.001465 seconds
gv1  | [2022.03.25-18.43.16:091][673]LogOnlineSession: Warning: STEAM: Server setting ,SNAME_s:Grokenvinter discord.gg/tCMNAhQzrb overflows Steam SetGameTags call
gv1  | [2022.03.25-18.43.16:091][673]LogOnlineSession: Warning: STEAM: Server setting ,ServerName_s:Grokenvinter discord.gg/tCMNAhQzrb overflows Steam SetGameTags call
gv1  | [2022.03.25-18.43.16:091][673]LogOnlineSession: Warning: STEAM: Server setting ,HasPassword_b:false overflows Steam SetGameTags call
gv1  | [2022.03.25-18.43.16:091][673]LogOnlineSession: Warning: STEAM: Server setting ,IsExperimental_b:false overflows Steam SetGameTags call
gv1  | [2022.03.25-18.43.16:091][673]LogOnline: Verbose: OSS: Async task 'FOnlineAsyncTaskSteamUpdateServer bWasSuccessful: 1 Session: GameSession' succeeded in 0.000770 seconds
gv1  | [2022.03.25-18.43.31:157][123]LogOnlineSession: Warning: STEAM: Server setting ,SNAME_s:Grokenvinter discord.gg/tCMNAhQzrb overflows Steam SetGameTags call
gv1  | [2022.03.25-18.43.31:157][123]LogOnlineSession: Warning: STEAM: Server setting ,ServerName_s:Grokenvinter discord.gg/tCMNAhQzrb overflows Steam SetGameTags call
gv1  | [2022.03.25-18.43.31:157][123]LogOnlineSession: Warning: STEAM: Server setting ,HasPassword_b:false overflows Steam SetGameTags call
gv1  | [2022.03.25-18.43.31:157][123]LogOnlineSession: Warning: STEAM: Server setting ,IsExperimental_b:false overflows Steam SetGameTags call
gv1  | [2022.03.25-18.43.31:157][123]LogOnline: Verbose: OSS: Async task 'FOnlineAsyncTaskSteamUpdateServer bWasSuccessful: 1 Session: GameSession' succeeded in 0.001199 seconds
gv1  | [2022.03.25-18.43.39:244][365]LogOnline: Verbose: STEAM: FOnlineAsyncEventSteamConnectionRequest RemoteId: UNKNOWN [0x7F29F359D2A0]
gv1  | [2022.03.25-18.43.39:244][365]LogOnline: STEAM: Adding P2P connection information with user 76561198208410509 (Name: UNKNOWN [0x7F29F392AC30])
gv1  | [2022.03.25-18.43.42:843][472]LogNet: Verbose: NotifyAcceptingConnection: Server LongvinterIsland accept
gv1  | [2022.03.25-18.43.42:843][472]LogNet: NotifyAcceptingConnection accepted from: 76561198208410509:7777
gv1  | [2022.03.25-18.43.42:843][472]LogNet: Verbose: NotifyAcceptingConnection: Server LongvinterIsland accept
gv1  | [2022.03.25-18.43.42:843][472]LogNet: NotifyAcceptingConnection accepted from: 76561198208410509:7777
gv1  | [2022.03.25-18.43.42:843][472]LogNet: Verbose: NotifyAcceptingConnection: Server LongvinterIsland accept
gv1  | [2022.03.25-18.43.42:843][472]LogNet: NotifyAcceptingConnection accepted from: 76561198208410509:7777
gv1  | [2022.03.25-18.43.42:843][472]LogNet: Verbose: NotifyAcceptingConnection: Server LongvinterIsland accept
gv1  | [2022.03.25-18.43.42:843][472]LogNet: NotifyAcceptingConnection accepted from: 76561198208410509:7777
gv1  | [2022.03.25-18.43.42:877][473]LogNet: Verbose: NotifyAcceptingConnection: Server LongvinterIsland accept
gv1  | [2022.03.25-18.43.42:878][473]LogNet: NotifyAcceptingConnection accepted from: 76561198208410509:7777
gv1  | [2022.03.25-18.43.42:879][473]LogNet: Server accepting post-challenge connection from: 76561198208410509:7777
gv1  | [2022.03.25-18.43.42:879][473]LogNet: Verbose: InitSequence: IncomingSequence: 13052, OutgoingSequence: 10002, InitInReliable: 764, InitOutReliable: 786
gv1  | [2022.03.25-18.43.42:879][473]PacketHandlerLog: Loaded PacketHandler component: Engine.EngineHandlerComponentFactory (StatelessConnectHandlerComponent)
gv1  | [2022.03.25-18.43.42:879][473]LogNet: Verbose: UNetConnection::SetClientLoginState: State changing from Invalid to LoggingIn
gv1  | [2022.03.25-18.43.42:879][473]LogNet: Verbose: UNetConnection::SetExpectedClientLoginMsgType: Type same: [0]Hello
gv1  | [2022.03.25-18.43.42:879][473]LogOnline: STEAM: Adding user 76561198208410509:7777 from RegisterConnection
gv1  | [2022.03.25-18.43.42:879][473]LogNet: NotifyAcceptedConnection: Name: LongvinterIsland, TimeStamp: 03/25/22 18:43:42, [UNetConnection] RemoteAddr: 76561198208410509:7777, Name: SteamNetConnection_2147436785, Driver: GameNetDriver SteamNetDriver_2147482569, IsServer: YES, PC: NULL, Owner: NULL, UniqueId: INVALID
gv1  | [2022.03.25-18.43.42:879][473]LogNet: AddClientConnection: Added client connection: [UNetConnection] RemoteAddr: 76561198208410509:7777, Name: SteamNetConnection_2147436785, Driver: GameNetDriver SteamNetDriver_2147482569, IsServer: YES, PC: NULL, Owner: NULL, UniqueId: INVALID
gv1  | [2022.03.25-18.43.42:958][474]LogNet: NotifyAcceptingChannel Control 0 server World /Game/Maps/LongvinterIsland.LongvinterIsland: Accepted
gv1  | [2022.03.25-18.43.42:958][474]LogNet: Remote platform little endian=1
gv1  | [2022.03.25-18.43.42:958][474]LogNet: This platform little endian=1
gv1  | [2022.03.25-18.43.42:958][474]LogNet: Verbose: UNetConnection::SetExpectedClientLoginMsgType: Type changing from [0]Hello to [5]Login
gv1  | [2022.03.25-18.43.42:991][475]LogNet: Login request: ?Name=grokenstein userId: Steam:UNKNOWN [0x7F29F3921360] platform: Steam
gv1  | [2022.03.25-18.43.42:992][475]LogNet: Verbose: UNetConnection::SetClientLoginState: State changing from LoggingIn to Welcomed
gv1  | [2022.03.25-18.43.43:608][493]LogNet: Verbose: UNetConnection::SetClientLoginState: State same: CleanedUp
gv1  | [2022.03.25-18.43.43:641][494]LogNet: Verbose: UNetConnection::SetClientLoginState: State same: CleanedUp
gv1  | [2022.03.25-18.43.43:994][505]LogNet: Client netspeed is 100000
gv1  | [2022.03.25-18.43.43:994][505]LogNet: Join request: /Game/Maps/LongvinterIsland?Name=grokenstein?SplitscreenCount=1
gv1  | [2022.03.25-18.43.43:995][505]r.Shadow.RadiusThreshold = "0.01"
gv1  | [2022.03.25-18.43.43:995][505]r.Shadow.RadiusThreshold = "0.01"
gv1  | [2022.03.25-18.43.43:995][505]r.Shadow.FadeResolution = "16"
gv1  | [2022.03.25-18.43.43:995][505]r.Shadow.MinResolution = "8"
gv1  | [2022.03.25-18.43.43:995][505]r.Shadow.TexelsPerPixel = "4"
gv1  | [2022.03.25-18.43.43:995][505]LogGameMode: FindPlayerStart: PATHS NOT DEFINED or NO PLAYERSTART with positive rating
gv1  | [2022.03.25-18.43.43:997][505]LogNet: Join succeeded: grokenstein
gv1  | [2022.03.25-18.43.43:997][505]LogNet: Verbose: UNetConnection::SetClientLoginState: State changing from Welcomed to ReceivedJoin
gv1  | [2022.03.25-18.43.44:529][521]LogNet: Server connection received: ActorChannelFailure [UChannel] ChIndex: 0, Closing: 0 [UNetConnection] RemoteAddr: 76561198208410509:7777, Name: SteamNetConnection_2147436785, Driver: GameNetDriver SteamNetDriver_2147482569, IsServer: YES, PC: PC_Longvinter_C_2147436781, Owner: PC_Longvinter_C_2147436781, UniqueId: Steam:UNKNOWN [0x7F29F4379700]
gv1  | [2022.03.25-18.43.44:529][521]LogNet: Warning: UControlChannel::ReceivedBunch: NetConnection::Close() [GameNetDriver] [PC_Longvinter_C_2147436781] [PC_Longvinter_C_2147436781] from failed to initialize the PlayerController channel. Closing connection.
gv1  | [2022.03.25-18.43.44:529][521]LogNet: UNetConnection::Close: [UNetConnection] RemoteAddr: 76561198208410509:7777, Name:
```

Says join succeeded. Then Welcomed. Then Welcomed to ReceivedJoin. Then `ActorChannelFailure` and something about `failed to initialize the PlayerController channel. Closing connection.`. Then `UNetConnection::Close`. And that's when I get kicked back to main menu in game.

**Comment by Cyprith** [comment]
I am also unable to access any servers

**Comment by jehoshua02** [comment]
To resolve the issue, 

1. I tried removing containers and images entirely, then up'ing container again. No luck.
2. Figuring it had something to do with old data being incompatible with new server version, I built a pristine docker volume by recreating the container with a new, empty volume mount.
3. Then I spun up a container with old and new volumes, and a noop entrypoint, exec in and copied `SaveGames` from old to new volume. Removed that container.
10. Then up the container again with the new volume.
11. Was able to connect to server and see character where I last left him.

So seems like it was something in the old volume that was incompatible with the new server version.
